### ISSUE TITLE
CDDL parsing fails with carefully placed newline.

### ISSUE THREAD

**Issue by d3zd3z** [anweiss/cddl#80]
In the following block of CDDL:

```cddl
top-level = top-group

;; This fails to parse with the newline after the "=>"
top-group //= (identifier-a =>
    int)

;; It parses without the newline, or if the newline is before the
;; arrow.
top-group //= (identifier-b => int)
top-group //= (identifier-c
    => int)

identifier-a = 1
identifier-b = 2
identifier-c = 3
```

The first `top-group` declaration cause a parse error. It seems to require both using an identifier as the key, and having the newline after the arrow. It is easy to work around by either moving the arrow to the new line, or just joining the lines.

```
error: parser errors
  ┌─ input:4:31
  │
4 │   top-group //= (identifier-a =>
  │ ╭──────────────────────────────^
5 │ │     int)
  │ ╰^ invalid group entry syntax

Error: "Parser error"
```

**Comment by anweiss** [comment]
Thanks for reporting @d3zd3z. Will get this fixed!

**Comment by anweiss** [comment]
This has been fixed as of [v0.8.6](https://github.com/anweiss/cddl/releases/tag/0.8.6). Pleas reopen if you are still experiencing issues.
